openapi.yaml

```diff
@@ -11,7 +11,7 @@ info:
     name: MIT
     url: "https://opensource.org/license/mit/"
 servers:
-  - url: "https://api.fake-museum-example.com/v1.1"
+  - url: "https://redocly.com/_mock/docs/openapi/museum-api"
```

Comment:
@RomanHotsiy / @lornajane 
Please check this out, we can revert it back.

This is controversial change: 
+ When spot is released and somebody wan't to check it out there will be a change to run successfully Arazzo tests
+ Using this API in other tool will allow to get mocked responses
- This will make our mockserver public for everyone, might bring some other spam issues

Reply:
I would change it or add it as an additional server.

Reply:
Ok, so if it is allowed to display our mock-server URL publicly - I would keep it with mock-server URL, as the other server leads nowhere.